llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -308,6 +308,11 @@ cl::opt<memprof::IndexedVersion> MemProfVersionRequested(
                clEnumValN(memprof::Version1, "1", "version 1"),
                clEnumValN(memprof::Version2, "2", "version 2")));
 
+cl::opt<bool> MemProfFullSchema("memprof-full-schema", cl::Hidden,
+                                cl::sub(MergeSubcommand),
+                                cl::desc("Serialize the full schema"),
```

Comment:
nit: "Use the full schema for serialization" -- The current statement sounds like it applies to the schema and not the content.

Reply:
Adressed in the latest iteration.